spec/features/budgets/access_index.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+feature "access index" do
+  scenario "successfully" do
+    user = create(:user)
+    log_in_as user
+    budget1 = create(:budget)
+    budget2 = create(:budget)
+
+    visit budgets_path
+
+    expect(page).to have_text(I18n.t("budgets.index.title"))
+    expect(page).to have_text(budget1.name)
+    expect(page).to have_text(budget2.name)
+    expect(Budget.all.count).to eq 2
```

Comment:
This is not checking something in the UI and does not belong in a feature spec.

Also a question: why are we testing this?

Reply:
Put like that, I don't know, it's gone now - sha: 1502d2e